### ISSUE TITLE
Add call animation

### ISSUE THREAD

**Issue by Benteltje** [Ebullioscopic/DynamicIsland#248]
Add call animation

like this: https://www.reddit.com/r/macapps/comments/1kq8i4v/dynamic_island_call_notifications_all_new_music/

**Comment by bitmap4** [comment]
Hi I thought I'd pick this up, but I can't figure out a way to hide the native call banner.

Setting facetime's alert style to none doesn't hide it, and moving the banner offscreen via accessibility doesn't work either since it seems to snap back immediately (or doesn't move at all?). I believe this is because the banner is drawn by `FaceTimeNotificationExtension` and doesn't show up as its own window (only `notificationcenter`'s host window does), so I can't hide by window id.

Any ideas from the devs?
